diagnostic.md

```diff
@@ -3,36 +3,44 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Low bandwith environments with single page applications.
+You can control state with API only development.
+Can do away with jquery.
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+The templating system is handlebars. You load templates and components based on URLs.
 ```
 
 Name at least two layers of an Ember application (1.0 or 2.0):
 
 ```sh
-# your answer here
+View
+Model
+View Model
 ```
 
 How does a component differ from an Ember template?
 
 ```sh
-# your answer here
+A template is made up of components. Componets are pre-determined sub sets of the templates that you want to change.
 ```
 
 What is an opinionated framework? Is Ember opinionated? How or how not?
 
 ```sh
-# your answer here
+An opinionated framework is software that strictly guides the user to do things
+the way they want. Ember is an opinionated framework because the route is thing that starts ebverything off
+and dictates what the user will be able to do.
 ```
 
 In Ember what is a route's job (please list three)?
 
 ```sh
-# your answer here
+Controll state of the application
+Call templates and components
+Sorts the model. 
```

Comment:
Correct! I would phrase them a little differently:
1. Parse information from the url
2. Link the route to a component or template
3. Load the template/UI data via the 'model' hook.

1/1